src/cramFile/util.js

```diff
@@ -141,7 +141,13 @@ module.exports = {
     const method = _class.prototype[methodName]
     const memoAttrName = `_memo_${methodName}`
     _class.prototype[methodName] = function _tinyMemoized() {
-      if (!(memoAttrName in this)) this[memoAttrName] = method.call(this)
+      if (!(memoAttrName in this)) {
+        this[memoAttrName] = method.call(this)
+        Promise.resolve(method.call(this)).catch(err => {
```

Comment:
This calls the method twice (`method.call` actually runs the code). Need to fix that

Reply:
I can't find this code in the "Files changed"...can't see it on the current this branch or on master, is it in one of the commits? I'll likely squash it out if it is